gravitee-apim-portal-webui-next/src/entities/api/api.ts

```diff
@@ -63,6 +63,7 @@ export interface Api {
    * Last update date and time.
    */
   updated_at?: Date;
+  security?: string;
```

Comment:
What about type this with a tuple like `ApiSecurity = 'JWT' | 'OAUTH2' ...` ?

Reply:
Done